.template/addons/github/.github/workflows/publish_wiki.yml.tt

```diff
@@ -2,17 +2,35 @@ name: Publish Wiki
 
 on:
   push:
-    paths:
-      - .github/wiki/**
     branches:
       - develop
+    paths:
+      - .github/wiki/**
+
+env:
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: bot@nimblehq.co
 
 jobs:
   publish:
-    name: Publish Wiki
-    uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0
-    with:
-      USER_NAME: github-wiki-workflow
-      USER_EMAIL: ${{ secrets.GH_EMAIL }}
-    secrets:
-      USER_TOKEN: ${{ secrets.GH_TOKEN }}
+    name: Publish Github Wiki
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
```

Comment:
could we use `concurrency` instead? There's a similar implementation in:
https://github.com/nimblehq/rails-templates/blob/46573c27d4cbce5dcb9c1f8d3ff2984256f44799/.github/workflows/test_generated_app.yml#L16-L19

Reply:
Thanks. Fixed inf68814abcb4b2b56cbc8a26fea64e0b865337766